drools-game-engine-services/src/test/java/org/drools/game/services/endpoint/test/CaptureTheFlagTest.java

```diff
@@ -0,0 +1,115 @@
+package org.drools.game.services.endpoint.test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+
+import org.drools.game.capture.flag.cmds.EnterZoneCommand;
+import org.drools.game.capture.flag.cmds.PickFlagCommand;
+import org.drools.game.capture.flag.model.Flag;
+import org.drools.game.capture.flag.model.Zone;
+import org.drools.game.core.api.StoreFacts;
+import org.drools.game.model.api.Player;
+import org.drools.game.model.impl.base.BasePlayerImpl;
+import org.drools.game.services.endpoint.api.GameService;
+import org.drools.game.services.endpoint.impl.CommandRest;
+import org.drools.game.services.endpoint.impl.GameServiceImpl;
+import org.drools.game.services.infos.GameSessionInfo;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.resteasy.client.jaxrs.ResteasyWebTarget;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.swarm.jaxrs.JAXRSArchive;
+
+@RunWith( Arquillian.class )
+public class CaptureTheFlagTest {
+
+	private Client client;
+    private ResteasyWebTarget target;
+    private static final String APP_URL = "http://localhost:8080/";
+    
+    @BeforeClass
+    public static void setupCmds() {
+    	// FIXME Relocated temporarily on StoreFactsTestImp
+    	
+//        CommandRegistry.set( "TELEPORT_CALLBACK", "org.drools.game.services.endpoint.test.cmds.TeleportPlayerCommand" );
```

Comment:
This also needs to be set by the configurations of the game on the server side. So all that you called StoreFacts** is related with the games configurations and needs to be abstracted, so the GameService when it is bootstrapped, picks it up from the game package.